HomeWork 4/HomeWork 4/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+class Program
+{
+    static void Main()
+    {
+        Console.Write("N = ");
+        int N = Convert.ToInt32(Console.ReadLine());
+        int[] arr = new int[N];
+        Console.WriteLine($"Enter {N} elements:");
+        for (int i = 0; i < N; i++)
+        {
+            Console.Write($"Element {i + 1}:");
+            arr[i] = Convert.ToInt32(Console.ReadLine());
+        }
+        Console.WriteLine();
+        Console.Write("Array elements: ");
+        foreach (int element in arr)
+        {
+            Console.Write(element + "; ");
+        }
+
+        Random rnd = new Random();
+
+        for (int i = 0; i < N; i++)
+        {
+            arr[i] = rnd.Next(1, 27);
+        }
+        Console.WriteLine();
+        Console.Write("Array elements: ");
+        foreach (int element in arr)
+        {
+            Console.Write(element + "; ");
+        }
+        Console.WriteLine();
+
+        Console.WriteLine("Odd numbers from 1 to 26: ");
+        for (int i = 1; i <= 26; i++)
+        {
+            if (i % 2 != 0)
+            {
+                Console.Write(i + "; ");
+            }
+        }
+        Console.WriteLine();
+
+        Console.WriteLine("Even numbers from 1 to 26: ");
+        for (int i = 1; i <= 26; i++)
+        {
+            if (i % 2 == 0)
+            {
+                Console.Write(i + "; ");
+            }
+        }
+        Console.Write("Enter the value of N: ");
+        if (int.TryParse(Console.ReadLine(), out N) && N > 0)
+        {
+            int[] evenArray = new int[N];
+            int[] oddArray = new int[N];
+
+            char[] evenLetters = new char[N];
+            char[] oddLetters = new char[N];
+
+            char[] lettersToUppercase = { 'a', 'e', 'i', 'd', 'h', 'j' };
+
+            Random random = new Random();
```

Comment:
We don't need to create new instance of Random here, we can use one that has been created on line 23